deepinv/physics/forward.py

```diff
@@ -346,6 +346,16 @@ def A_adjoint(self, y):
 
         return self.A_adj(y)
 
+    def A_grad(self, diff, x_est):
+        r"""
+        Computes the gradient of the forward operator :math:`\nabla_x f(A(x))`. It can make the representation of the gradient of the L2 loss more general and compact.
```

Comment:
I think there is a confusion here.

This function should compute a jacobian vector product between $\frac{\delta A}{\delta x} v$. where $v$ is your `diff` input and the jacobian $\frac{\delta A}{\delta x}$ is evaluated at `x_est`.

I would change the name of `diff` for `v` or `u` and also change the name of `x_est` for just `x`.